CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## [Unreleased]
 
+### Fixed
+- Typo when importing `chalk`
+
```

Comment:
```suggestion
```

Reply:
Nice point, thanks! `## [1.13.1] - 2021-05-25` was released before the finish of the pull request. I'm planning to do a new release to include new changes on NPM. Without add this change, there will be a blank release on the repo. Make sense for you?

Reply:
Yeah, the tag has indeed been created before it should've been. You'll have to make a new release, in fact. Make sure you specify in the CHANGELOG, under the section `## [1.13.1] - 2021-05-25`, the changes that have already been applied. Then you can announce the new changes in a new section.